test/events.js

```diff
@@ -0,0 +1,95 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const assert = require('insist');
+const crypto = require('crypto');
+const fs = require('fs');
+const path = require('path');
+
+const db = require('../lib/db');
+const events = require('../lib/events');
+const P = require('../lib/promise');
+
+const UID = crypto.randomBytes(16).toString('hex');
+const mock = require('./lib/mock')({ userid: UID });
+const Server = require('./lib/server');
+const Static = require('./lib/static');
+
+const imagePath = path.join(__dirname, 'lib', 'firefox.png');
+const imageData = fs.readFileSync(imagePath);
+
+const SIZES = require('../lib/img').SIZES;
+const SIZE_SUFFIXES = Object.keys(SIZES).map(function(val) {
+  if (val === 'default') {
+    return '';
+  }
+  return '_' + val;
+});
+
+
+/*global describe,it,before*/
+
+describe('events', function() {
+  describe.only('onDeleteMessage', function() {
+    var tok = crypto.randomBytes(32).toString('hex');
+    function Message(del) {
+      return {
+        event: 'delete',
+        uid: UID + '@accounts.firefox.com',
+        del: del
+      };
+    }
+
+    before(function() {
+      this.slow(2000);
```

Comment:
Ah, I copied this from the api test. Uploading was the first thing I did in this repo, and the tests used to be slow, until I added the `mock` stuff. I must have forgotten to take these out. Killed.